changelogs/fragments/openssl_privatekey-generate.yaml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+  - openssl_privatekey can't create files which only have read permissions #48656(https://github.com/ansible/ansible/issues/48656)
+  - Overwrite the symbolic link with a file only when the force option is specified
```

Comment:
Hi! Thank you for the review!
In the current implementation, when a symbolic link is overwritten as a file, and the original file at the link destination will not be overwritten and will remain intact. I'm thinking that when we executed this module without force option, it would to be better to fail the overwrite process.
If my idea is okay, I will append this behavior to the document.

Reply:
Sure! I will do this! thanks!